packages/eslint-plugin-pf-codemods/src/rules/v6/toolbarReplacedSpacerSpaceItems/toolbar-replaced-spacer-spaceItems.ts

```diff
@@ -0,0 +1,73 @@
+import { Rule } from "eslint";
+import { JSXOpeningElement, JSXAttribute } from "estree-jsx";
+import { Property } from "estree-jsx";
+import { getFromPackage, getAttribute, getAttributeValue } from "../../helpers";
+
+// https://github.com/patternfly/patternfly-react/pull/10418
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context: Rule.RuleContext) {
+    const { imports } = getFromPackage(context, "@patternfly/react-core");
+
+    const toolbarImports = imports.filter((specifier) =>
+      ["ToolbarGroup", "ToolbarToggleGroup", "ToolbarItem"].includes(
+        specifier.imported.name
+      )
+    );
+
+    return !toolbarImports.length
+      ? {}
+      : {
+          JSXOpeningElement(node: JSXOpeningElement) {
+            if (
+              node.name.type === "JSXIdentifier" &&
+              toolbarImports
+                .map((imp) => imp.local.name)
+                .includes(node.name.name)
+            ) {
+              const spacerProp = getAttribute(node, "spacer");
+              const spaceItemsProp = getAttribute(node, "spaceItems");
+              if (!spacerProp && !spaceItemsProp) {
+                return;
+              }
+              const spacerPropMessage = `The spacer property has been removed from ${node.name.name}. We recommend instead using our new gap, columnGap, or rowGap properties.`;
+              const spaceItemsPropMessage = `${
+                spacerProp ? " Additionally, the" : "The"
+              } spaceItems property has been removed from ${node.name.name}.`;
+              const spacerVal =
+                spacerProp && getAttributeValue(context, spacerProp.value);
+
+              context.report({
+                node,
+                message: `${spacerProp ? spacerPropMessage : ""}${
+                  spaceItemsProp ? spaceItemsPropMessage : ""
+                }`,
+                fix(fixer) {
+                  const fixes = [];
+
+                  if (spacerProp) {
+                    spacerVal &&
+                      spacerVal.forEach((val: Property) => {
+                        const newValue =
```

Comment:
`newValue` can be `false` here, which could be a problem in a use case like this:

```
const NO_SPACER = "spacerNone";
<ToolbarItem spacer={{ default: NO_SPACER }} />
```
which results to:
```
const NO_SPACER = "spacerNone";
<ToolbarItem spacer={{ default: "false" }} />
```

I don't think we need to attempt to change the variable value to `gapNone`, because this is a very rare use case and probably nobody uses it like this. But I think the variable name should stay there so we don't mess up with consumer's code just in case.